tmux-gitbar.tmux

```diff
@@ -3,15 +3,22 @@
 # Created by Aurélien Rainone
 # github.com/aurelien-rainone/tmux-gitbar
 
-# generate tmux-gitbar.conf if it doesn't exist
-if-shell 'test -z ${TMUX_GITBAR_DIR}' \
-  "if-shell '! stat ~/.tmux-gitbar/tmux-gitbar.conf' \
-    'run-shell \"~/.tmux-gitbar/lib/generate-config.sh ~/.tmux-gitbar/tmux-gitbar.conf\"'" \
-  "if-shell '! stat $TMUX_GITBAR_DIR/tmux-gitbar.conf' \
-    'run-shell \"$TMUX_GITBAR_DIR/lib/generate-config.sh $TMUX_GITBAR_DIR/tmux-gitbar.conf\"'"
+# Set defaults
+# FIXME: Why can I only get this to work if I double nest?
+if-shell 'test True' \
+  "if-shell 'test -z \"${TMUX_GITBAR_DIR}\"' \
+    'TMUX_GITBAR_CONF=\"$HOME/.tmux-gitbar.conf\"' \
+    'TMUX_GITBAR_CONF=\"$TMUX_GITBAR_CONF\"'"
```

Comment:
The pseudo code i have in mind for `tmux-gitbar.tmux` too bad it involves a new level of nesting... :

```python

""" 1st step: generate configuration """
if "$tmgb_conf" env var is set:
	if "$tmgb_conf" file not exist:
		if "$tmgb_dir" envvar is set:	
			"$tmgb_dir/lib/generate_config.sh" $(tmgb_conf)
		else:
			"$home/.tmgb/lib/generate_config.sh" $(tmgb_conf)
else:
	if "$tmgb_dir" envvar is set:	
		"$tmgb_dir/lib/generate_config.sh" ~/.tmgb.conf
	else:
		"$home/.tmgb/lib/generate_config.sh" ~/.tmgb.conf

""" 2nd step: install prompt_command
(THIS SHOULDNT BE DIFFERENT THAN CURRENT VERSION"""
if update-gitbar not in $prompt_cmd:
	if "$tmgb_dir" is set:
		"$prompt_cmd" += "$tmgb_dir/update-gitbar"
	else:
		"$prompt_cmd" += "~/.tmgb/update-gitbar"
```